README.md

```diff
@@ -29,52 +28,54 @@
 
 ## What is Karpor?
 
-Karpor is a Modern **Kubernetes Explorer** focusing on **🔍 Search**, **💡 Insight** and **🤖 Intelligence**. It has keywords such as `self-host`, `non-invasive`, `read-only`, `secure compliance`, `multi-cloud/multi-cluster support`, `custom logical view`, and can serve as a **Kubernetes Data Plane** to reduce the cost of discovering and understanding kubernetes resources.
+Karpor is Intelligence for Kubernetes. It brings advanced **🔍 Search**, **💡 Insight** and **✨ AI** to Kubernetes. With Karpor, you can gain more visibility into your Kubernetes clusters across any clouds and empower developers to ship faster.
+
+We hope to become a **small and beautiful, vendor-neutral, community-driven** project! 🚀
 
 https://github.com/KusionStack/karpor/assets/9360247/c5050dfa-23f3-49ac-ba4a-1026ab043e6c
 
 ## Why Karpor?
 
-<h3 align="center">🤝 User Friendly</h3>
+<h3 align="center">🔍 Search</h3>
 
 <table>
   <tr>
-    <td style="vertical-align: middle;">
-      <b>⚡️ Lightweight and Easy to Setup</b><br />One `helm` is done.<br /><br/>
-      <b>📦 Out of the Box</b><br />Built-in security and compliance policies, resource sync strategy, resource transform rule, and topology relationship definitions.<br /><br />
-      <b>🔍 Quickly search and locate resource(s)</b><br />Quickly search and locate resource(s) of interest across a large number of clusters in a user-friendly way.
+    <td>
+      <b>⚡️ Automatic Syncing</b><br />Automatically synchronize your resources across any clusters managed by the multi-cloud platform.<br /><br/>
```

Comment:
The aspect ratio of the text and image is not very harmonious.

Reply:
Yes, I also found this problem, but in GitHub Markdown, it does not support CSS syntax such as style, width ... (Set up and not work)